setup.py

```diff
@@ -986,7 +986,6 @@ def debug_build():
 
 try:
     setup(
-        version=PILLOW_VERSION,
```

Comment:
Is there a way to query the version from setuptools instead of reading it from a file in `get_version`?

Reply:
There are 7 options at https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

1. Also reads from file
2. I'm not too sure about those external build tools
3. We're using something like this
4. Also reads from file
5. Moves version constant to `setup.py`, project code fetchs via `importlib.metadata`
6. Import the version from `src/PIL/__init__.py`, but I'm not sure if we can import PIL before the build?
7. Get the version number from SCM using setuptools_scm. I've used setuptools_scm in other projects, this might work.

But I suggest these could be topics for a followup.